### DISCUSSION TITLE
Why does shadcn-ui init adds `container` to Tailwind config?

### DISCUSSION THREAD

**Discussion by inwardmovement** [shadcn-ui/ui#2066] [Q&A] [answered]
Why does shadcn init adds this to `tailwind.config.ts`?

```
theme: {
  container: {
    center: true,
    padding: "2rem",
    screens: {
      "2xl": "1400px",
    },
  },
```

**Comment by onurhan1337** [comment, marked as answer]
After seeing the your discussion, I browsed around the project a little and saw that ``container` was not used in any component except the document page. In this way, I published a commit that removed the unused container from the necessary places. Thanks for your attention.

https://github.com/shadcn-ui/ui/pull/2085
